reports/registeredreport.qmd

```diff
@@ -8,22 +8,22 @@ csl: nature.csl
+Open science practices, such as publishing data and code alongside manuscripts, increase transparency and improve efficiency [@lowndes2017; @hampton2017]. The "openness" of data is determined by multiple factors. The FAIR principles [@wilkinson2016] - findability, accessibility, interoperability, and reusability - provide a framework for facilitating open data sharing. FAIR data are a critical component of groundbreaking synthesis science [@halpern2020; @fredston2023] and effective conservation practices [@roche2021; @buxton2021]. **NOTE**[I'd like to incorporate Scarpignato et al. 2023 [@scarpignato2023] here. SD: see suggestions below.]{.aside}. Moreover, for fields like ecology and geosciences where a lack of diversity reflects disproportionately hostile treatment of excluded groups [@primack2023; @primack2023; @marín-spiotta2020], FAIR data can serve an important role in transformations towards inclusive research cultures [@ledgerwood2022; @graves2022] (a transformation the open science movement itself is currently undertaking [@srivastava2014; @whitaker2020]). **NOTE**[In the context of movement ecology I often hear a somewhat conflicting argument, i.e., that those already advantaged take less risk and have greater potential for reward through sharing and using open data, e.g. journals with resources to support strict data-sharing policies, organizations with the expertise and computing infrastructure to mine data and build profitable applications, universities with the mission and capacity to run meta-analyses and extract knowledge from their own data and the data of others. In contrast, where data that are findable but not open (less FAIR), those with less access and connections may disproportionately benefit from the direct contact required to discuss and agree on data sharing when data cannot be freely downloaded, which can promote new networking, trust-building, and appropriate participation and credit. Rather than dig into this here, a more direct argument for open data in the context of this study/field might reference Scarpignato et al., the importance of geographic/taxonomic representation in open data, and the ethical argument to ensure data access for future generations (also see my note in \[issue #12\](https://github.com/FlukeAndFeather/openbiologging/issues/12).]{.aside}
```

Comment:
the ethics of open data are certainly complex and i want to tread carefully here. the concern about open data requirements benefiting wealthy institutions and taking away opportunities from less-resourced institutions is valid. the debate over open _access_, where the objections of scientists from the global south about prohibitive apc's were largely ignored, is relevant here. at the same time, data hoarding by scientists at wealthy institutions is a real phenomenon, and plenty of PIs use access to their (often publicly-funded) data as leverage for unethical practices, from inflated co-authorship to abusive behavior. I suspect non-FINDable data actually disproportionately benefits the already-connected and already-resourced, evidenced by the large number of high-impact papers with "old boys club" author lists. i would really like to discuss this further with the whole author list in a facilitated conversation!

The point you bring up in issue #12 is something we absolutely have to address in the discussion.

Reply:
I like the idea of a discussion and certainly ethical considerations are valid. Given the nuance and because the analysis doesn't address this topic directly, for the introduction, I think it will be clearest to readers to focus on the most clear benefits of open data (e.g. those listed already as well as documenting biodiversity for future generations, and as you mention allowing access to publicly-funded work) to guide readers to the hypotheses and scope of the analysis.

Reply:
I've been going back and forth on this all morning, but I think you're right. I'll edit the introduction accordingly.